scripts/ci/verify_required_checks.mjs

```
@@ -0,0 +1,74 @@
+  // Fallback to API fetch if needed or if we want to be sure
+  if (present.size === 0 && GITHUB_TOKEN && GITHUB_REPOSITORY && GITHUB_SHA) {
+    const [owner, repo] = GITHUB_REPOSITORY.split("/");
+    const data = await gh(`/repos/${owner}/${repo}/commits/${GITHUB_SHA}/check-runs`);
+    (data.check_runs || []).forEach(c => present.add(c.name));
+  }
+
+  let missing = checks.filter(c => !present.has(c));
+
+  if (missing.length) {
+    console.error("❌ Missing required checks:");
+    missing.forEach(c => console.error(` - ${c}`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)